sdk/src/main/java/io/dapr/client/DaprHttpBuilder.java

```diff
@@ -26,26 +26,6 @@
    */
   private static final Object LOCK = new Object();
 
-  /**
-   * Read timeout used to build object.
-   */
-  private Duration readTimeout = Duration.ofSeconds(Properties.HTTP_CLIENT_READ_TIMEOUT_SECONDS.get());
-
-  /**
-   * Sets the read timeout duration for the instance to be built.
-   *
-   * <p>Instead, set environment variable "DAPR_HTTP_CLIENT_READTIMEOUTSECONDS",
-   *   or system property "dapr.http.client.readtimeoutseconds".
-   *
-   * @param duration Read timeout duration.
-   * @return Same builder instance.
-   */
-  @Deprecated
```

Comment:
quick question ... why was this deprecated?

Reply:
You cannot really set this per instance since the instance is singleton.